components/region_cache_memory_engine/src/gc.rs

```diff
@@ -0,0 +1,481 @@
+// Copyright 2023 TiKV Project Authors. Licensed under Apache-2.0.
+
+use core::slice::SlicePattern;
+use std::{fmt::Display, sync::Arc};
+
+use engine_traits::{CF_DEFAULT, CF_WRITE};
+use skiplist_rs::Skiplist;
+use slog_global::{info, warn};
+use txn_types::{Key, WriteRef, WriteType};
+
+use crate::{
+    keys::{decode_key, encoding_for_filter, InternalKey, InternalKeyComparator},
+    RegionCacheMemoryEngine,
+};
+
+fn split_ts(key: &[u8]) -> Result<(&[u8], u64), String> {
+    match Key::split_on_ts_for(key) {
```

Comment:
'split' in tikv context can be thought as it's region split, which is incorrect here. 
I think `split_on_ts_for` can be renamed as extract_key_ts

Reply:
`split_ts` follows the name in WriteCompactionFilter. Need we change it on both places as well as all places using `split_on_ts_for`.